add(bot): config merge.message.include_pull_request_author (#301)
chdsbd/kodiak: docs/docs/config-reference.md

This option only applies when `merge.message.body_type = "markdown"`.

### `merge.message.include_pull_request_author`

- **type:** `boolean`
- **default:** `false`

Add the pull request author as a coauthor of the merge commit using `Co-authored-by: jdoe <828352+jdoe@users.noreply.github.com>` syntax.

> This setting will override `merge.message.body = "github_default"` and `merge.message.body = "empty"`. In both cases the result will be an empty merge commit body with coauthor information at the end of the commit body.

Reviewer: ```suggestion
This setting will override `merge.message.body = "github_default"` and `merge.message.body = "empty"`. In both cases the result will be a commit message only container coauthor information.
```
